server/src/main/java/edu/jhu/server/Board.java

```diff
@@ -63,7 +63,7 @@ public Board() {
     }
 
     String[] suspects = {"colonel_mustard", "miss_scarlet", "professor_plum", "mr_green",
-        "mrs_white", "mrs_peacock"};
+        "mrs_white", "Mrs. Peacock"};
```

Comment:
I was going to ask about changing all of them at the meeting, but then I forgot. The alternative to changing them is to map proper names to server-side names on the client side.